{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "cc91eedc_da03ad3b",
        "filename": "/COMMIT_MSG",
        "patchSetId": 14
      },
      "lineNbr": 20,
      "author": {
        "id": 1072905
      },
      "writtenOn": "2023-03-23T08:41:18Z",
      "side": 1,
      "message": "As discussed during code review, this might or might not be the case, depending on the nature of the failure, perhaps this description should be updated accordingly?",
      "range": {
        "startLine": 19,
        "startChar": 16,
        "endLine": 20,
        "endChar": 22
      },
      "revId": "f0b930de264288984015a66deb207fc2a3cdf5ec",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153"
    }
  ]
}